velox/docs/functions/spark/struct.rst

```diff
@@ -0,0 +1,9 @@
+====================================
+Struct Functions
+====================================
+
+.. spark:function:: named_struct(name1, val1, name2, val2, ..., namen, valn) -> struct
```

Comment:
I'm not sure this "function" takes arguments like this. I also don't even think this works at all as I see logic in velox/expression/ExprCompiler.cpp that works only for "row_constructor".

```
const char* const kRowConstructor = "row_constructor";
...
ExprPtr getSpecialForm(
    const std::string& name,
    const TypePtr& type,
    std::vector<ExprPtr>&& compiledChildren,
    bool trackCpuUsage) {
  if (name == kRowConstructor) {
    return getRowConstructorExpr(
        type, std::move(compiledChildren), trackCpuUsage);
  }
```

It would be nice to add a unit test to confirm that "named_struct" works.

Reply:
Ok. I will double check it.

Reply:
Yes, you are right. velox just register [here](https://github.com/facebookincubator/velox/blob/main/velox/functions/sparksql/Register.cpp#L56). But UT can not pass. I removed this named_struct document.

Reply:
@zhejiangxiaomai Thank you for checking. Let's mention that named_struct function was removed in the PR description.